Tests/TestCountCheckTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Xunit;
+using FluentAssertions;
+using YoCode;
+
+namespace YoCode_XUnit
+{
+    public class TestCountCheckTests
+    {
+        private readonly string testStatLine;
+        private readonly string testTestOutput;
+
+        public TestCountCheckTests()
+        {
+            testTestOutput = "Build started, please wait...\n" +
+            "Build started, please wait...\n" +
+            "Build completed.\n" +
+            "Test run for)\n" +
+            "Microsoft(R) Test Execution Command Line Tool Version 15.7.0\n" +
+            "Copyright(c) Microsoft Corporation.All rights reserved.\n" +
+            "Starting test execution, please wait...\n" +
+            "Build completed.\n" +
+            "Test run for \n" +
+            "Microsoft(R) Test Execution Command Line Tool Version 15.7.0\n" +
+            "Copyright(c) Microsoft Corporation.All rights reserved.\n" +
+            "Starting test execution, please wait...\n" +
+            "Total tests: 22.Passed: 22.Failed: 0.Skipped: 0.\n" +
+            "Test Run Successful.\n" +
+            "Test execution time: 1.6349 Seconds\n";
+            testStatLine = "Total tests: 22.Passed: 22.Failed: 0.Skipped: 0.";
+
```

Comment:
If the point of this is to replicate what `dotnet test` outputs, I think there's a small mistake. There should be some spaces in the stats line: `"Total tests: 22. Passed: 22. Failed: 0. Skipped: 0."` It doesn't really matter though, the test is just that it matches what is in `testTestOutput`.

Reply:
You can use @ to start a multi-line string literal. e.g. 
```C#
testTestOutput = @"Build started, please wait...
Build completed.
Test run for)
Microsoft(R) Test Execution Command Line Tool Version 15.7.0
Copyright(c) Microsoft Corporation.All rights reserved.
Starting test execution, please wait...
Build completed.
Test run for 
Microsoft(R) Test Execution Command Line Tool Version 15.7.0
Copyright(c) Microsoft Corporation.All rights reserved.
Starting test execution, please wait...
Total tests: 22.Passed: 22.Failed: 0.Skipped: 0.
Test Run Successful.
Test execution time: 1.6349 Seconds";
```

Also, this code in `TestCountCheckTests.cs` has the line "Build started, please wait." twice.